### PR TITLE
Donations

### PR DESCRIPTION
**Pull request by Protagonistics**
DO EEEET

### CODE REVIEW

File: _posts/2015-06-29-donations.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+author: Adam Longwill
+---
+
+{:.img}
+![Donate your WiFi equipment to Meta Mesh!](http://i.imgur.com/HY0Hrpy.jpg)
+
+If you're like us, you probably have some old computer equipment collecting cobwebs in your basement. Technology "graveyards"
+are a surefire symptom of being chronically sick with IT-itis, which is, of course, incurable.
+
+Fortunately Meta Mesh is anouncing a donation program that will allow you to donate your old wireless
```
Comment (colindean) [inline _posts/2015-06-29-donations.md]: announcing

Comment (colindean) [inline _posts/2015-06-29-donations.md]: > is announcing

maybe "has a donation program now"

File: _posts/2015-06-29-donations.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+radios, routers, antennas, cables, and tools to Meta Mesh so that they can be given away for FREE to node hosts on PittMesh. Even
+though Meta Mesh's equipment is relatively inexpensive compared to many other wireless solutions, the cost to buy, configure, 
+and mount the equipment can still be prohibitive. Why not take up a collection and reuse old wireless equipment that is compatible with our routers?
+
+Here's a few details:
+    * Meta Mesh is not a 501c3 organization and therefore, your donations are NOT tax deductible.
```
Comment (colindean) [inline _posts/2015-06-29-donations.md]: 501(c)3 charitable organization

Comment (colindean) [inline _posts/2015-06-29-donations.md]: maybe add in here something like "if you want to target your donation to a specific charity operating in a neighborhood or municipality in which you would like to see PittMesh have a presence, we can work with you to make that happen"

File: _posts/2015-06-29-donations.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    situations where a potential PittMesh node host may be willing to host equipment but not comfortably able to afford the cost of 
+    certain elements of the installation.
+    * Meta Mesh reserves the right not to accept any equipment or to re-donate or discard any equipment for any reason.
+    * Meta Mesh may add branding for PittMesh (but not Meta Mesh) to further advertise the free and open network we are encouraging.
+    * Meta Mesh will refuse any equipment dropped off without our knowledge at our office. You must contact Meta Mesh first before donating any equipment.
+    
+We are most interested in the following items:
+  * 2.4 and 5GHz Antennas:
+    * Yagis
+    * > 10 DBi omni-directionals (NO "rubber ducky" antennas)
```
Comment (colindean) [inline _posts/2015-06-29-donations.md]: link to a picture of a rubber ducky antenna

File: _posts/2015-06-29-donations.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    * Sectors
+    * Biquads
+    * Parabolic Dishes (NO satellite dishes. Yes, they can be used for WiFi. No, we will not accept them)
+    * Patch and Panels (Preferably with N-Type connectors)
+    * Lightning arrestors or grounding equipment
+    
+  * Routers and switches
+    * 802.11n or 802.11ac routers only (Please do not donate your WRT54g's, as much as we love them)
+    * Preferably routers supported by Commotion. You can find the current list of compatible routers [here](https://commotionwireless.net/docs/supported-devices/)
+    * Switches only if they are gigabit speeds AND have a power adapter included.
```
Comment (colindean) [inline _posts/2015-06-29-donations.md]: or use a standard three-prong IEC 60320 C13 coupler power cable.

link to https://en.wikipedia.org/wiki/IEC_60320#C13.2FC14_coupler

these cables are a dime a dozen and a lot of enterprise switches use them

File: _posts/2015-06-29-donations.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    * Routers and switches in good, working condition only. Stained, cracked, broken, bricked, or "for parts only" routers and switches will not be accepted.
+    * Power over Ethernet injectors
+    
+  * Cables
+    * Unused indoor-rated (UTP, preferably plenum), Category 5e Ethernet cable in lengths over 100 feet, either terminated or unterminated.
+    * Unused outdoor-rated (STP), Category 5e Ethernet cable in any lengths, terminated or unterminated.
+    * Unused Category 5e RJ45 Ethernet ends of both the shielded and unshielded varieties.
+    * Gently used serial-USB cables, JTAG devices and cables, and pinout connectors (must come with documentation on their voltages)
+
```
Comment (colindean) [inline _posts/2015-06-29-donations.md]: maybe add bitcoin mining equipment?

lol 1/2 j/k

**Comment by Protagonistics** [comment]
Changes made per your suggestions. Thank you for playing! NOW PUSH IT REAAAL GOOD!

**Comment by Protagonistics** [comment]
Please push this to staging.

**Pull request merged**